pytrendy/io/plot_pytrendy.py

```
@@ -79,17 +112,23 @@ def plot_pytrendy(df: pd.DataFrame, value_col: str, segments_enhanced: list[dict
                     mask = (df.index >= prev_end) & (df.index <= prev_new_end) 
                     prev_color = color_map.get(segments_enhanced[i-1]['direction'], 'gray')
                     ax.fill_between(df.index[mask], ymin, ymax, color=prev_color, alpha=0.4)
 
-
         # Adjust ends when appropriate
         if (next_seg_abrupt or next_seg_noise) and next_neighbouring:
-            new_end = end + pd.Timedelta(days=1)
+            if index_type == 'date':
+                new_end = end + pd.Timedelta(days=1)
+            else:
+                new_end = end + 1
```

Ruff (0.15.20)

[warning] Use ternary operator `new_end = end + pd.Timedelta(days=1) if index_type == 'date' else end + 1` instead of `if`-`else`-block

Replace `if`-`else`-block with `new_end = end + pd.Timedelta(days=1) if index_type == 'date' else end + 1`

(SIM108)